node/sc/multi_bridge_test.go

```diff
@@ -1006,3 +1006,117 @@ func TestMultiBridgeKLAYTransferMixConfig3(t *testing.T) {
 		}
 	}
 }
+
+// TestNoncesAndBlockNumber checks the following:
+// - set threshold to 2.
+// - the first transfer is done and check nonces and block number (req 0, blk 100000)
+// - the second transfer is done and check nonces and block number (req 1, blk 100100)
+func TestNoncesAndBlockNumber(t *testing.T) {
```

Comment:
Reverted sequence case can be added as a unit test. :)

Reply:
There is a TestNoncesAndBlockNumberUnordered case.